### PR TITLE
feat(INT-53): bring in tf module from existing repo

### PR DESCRIPTION
**Pull request by westonplatter**
## what

- bring in existing postgres module under new repo name

## Summary by CodeRabbit

- **New Features**
  - Introduced comprehensive support for managing Google Workspace users, groups, group settings, and group memberships via Terraform.
  - Added detailed input variables for users and groups with extensive validation.
  - Provided example configurations and import workflows for existing organizations using YAML and Terraform files.

- **Documentation**
  - Rewrote and expanded the README with Google Workspace-specific usage instructions, examples, and input schemas.
  - Updated provider and resource documentation to reflect new functionality.

- **Bug Fixes**
  - Improved input validation for user and group attributes, ensuring correct email formats and allowed values.

- **Tests**
  - Added extensive test coverage for user and group variable validation, including edge cases and failure scenarios.

- **Chores**
  - Updated .gitignore rules and removed outdated changelog entries.
  - Removed obsolete outputs and variables related to previous random resource usage.

### CODE REVIEW

**Review by gberenice — APPROVED** [review]
🚢 🚢 🚢

File: examples/import-existing-org/imports.tf [hunk 1]
```diff
@@ -0,0 +1,61 @@
+locals {
+  # Locals specific to imports
+
+  # While the terraform-googleworkspace-users-groups-autogen module nests
+  # group settings in the group object, we need to access the group
+  # settings to import them into the provider's tf "group_settings" resource
+  group_settings = { for k, v in local._all_groups : k => merge(v.settings, { email = v.email }) if !startswith(k, "_") }
```
[the remark this answers is not in the document]
Comment (gberenice) [reply examples/import-existing-org/imports.tf:7]: It is defined in `examples/import-existing-org/main.tf`
